spec/helpers/enum_i18n_helper_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe EnumI18nHelper, type: :helper do
+  describe '#enum_options_for_select' do
+    subject { helper.enum_options_for_select(Service, :service_types) }
+
+    let(:expected_options) do
+      [
+        [t('activerecord.enums.service.service_types.normal'), 'normal'],
+        [t('activerecord.enums.service.service_types.long'), 'long'],
+        [t('activerecord.enums.service.service_types.probation'), 'probation']
+      ]
+    end
+
+    it { is_expected.to eq expected_options }
```

Comment:
If it's only one spec which is not too complicated, i'd prefer in lining the expectation instead of having a let. This would be easier to read since I do not need to go back, what the expected options actually are